notebooks/maze_tuto.ipynb

```diff
@@ -0,0 +1,642 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Maze tutorial\n",
+    "\n",
+    "In this tutorial, we tackle the maze problem.\n",
+    "We use this classical game to demonstrate how \n",
+    "- a new scikit-decide domain can be easily created.\n",
+    "- to find solvers from scikit-decide hub matching its characteristics\n",
+    "- to apply a scikit-decide solver to a domain\n",
+    "- to create its own rollout function to play a trained solver on a domain\n",
+    "\n",
+    "\n",
+    "Notes:\n",
+    "- In order to focus on scikit-decide use, we put some code not directly related to the library in a [separate module](./maze_utils.py) (like maze generation and display).\n",
+    "- A similar maze domain is already defined in [scikit-decide hub](https://github.com/airbus/scikit-decide/blob/master/skdecide/hub/domain/maze/maze.py) but we do not use it for the sake of this tutorial.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import io\n",
+    "from enum import Enum\n",
+    "from typing import NamedTuple, Optional, Any, List\n",
+    "from time import sleep\n",
+    "from collections import deque\n",
+    "import random\n",
+    "from math import sqrt, ceil\n",
+    "\n",
+    "from IPython.display import Markdown, display, clear_output\n",
```

Comment:
Sanitize imports - Markdown, plt ... are not used. To name a few.

Reply:
I just pushed a commit to #53 to sanitize notebooks too.